daemon/k8s_watcher.go

```diff
@@ -84,6 +103,10 @@ func (d *Daemon) EnableK8sWatcher(reSyncPeriod time.Duration) error {
 		return nil
 	}
 
+	if err := d.createThirdPartyResources(); err != nil {
+		return fmt.Errorf("Unable to create third party resource: %s", err)
```

Comment:
What happens if the api-server is not available when cilium starts? Won't this bail out without the user noticing?